README.md

```diff
@@ -8,7 +8,6 @@ If you're a developer wanting to contribute, read on.
 1. Make sure you've installed [Android Studio](https://developer.android.com/studio).
 1. Install npm using [Node Version Manager](https://github.com/nvm-sh/nvm)(nvm), as described in step one from the [Block Editor Quickstart guide](https://developer.wordpress.org/block-editor/getting-started/devenv/#quickstart)
 1. `cd WordPress-Android` to enter the working directory.
-1. `cp gradle.properties-example gradle.properties` to set up the sample app credentials file.
```

Comment:
Suggestion (💡): I am seeing this section a bit outdated/unnecessary, give-or-take the `npm` instruction. Wdyt if we go ahead and remove it, or replace it with just the `npm` instruction? 🤔

Reply:
I'd leave modifying this instruction to another PR: this already introduces multiple changes, I'd prefer to keep this concise.

Reply:
Not that we are at it, maybe we should at least change the `JDK8/11` reference on step 6... 🤷 😊